modules/ztp-image-based-upgrade-installing-lcao-with-gitops.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-installing-lcao-with-gitops_{context}"]
+= Installing the {lcao} with GitOps ZTP
+
+Install the {lcao} with GitOps ZTP to do an image-based upgrade.
```

Comment:
There's a couple common attributes you could use:
```
:ztp-first: GitOps Zero Touch Provisioning (ZTP)
:ztp: GitOps ZTP
```
e.g.:
```suggestion
= Installing the {lcao} with {ztp}

Install the {lcao} with {ztp-first} to do an image-based upgrade.
```
And another similar spot later.